docs/deploy/deploy-cic-openshift.md

```diff
@@ -125,69 +125,72 @@ Perform the following steps to deploy Citrix ADC CPX as a router with the Citrix
           name: citrix
           namespace: default
         ---
-        apiVersion: extensions/v1beta1
+        apiVersion: apps/v1
         kind: Deployment
         metadata:
+          labels:
+            app: cpx-cic
           name: cpx-cic
         spec:
           replicas: 1
+          selector:
+            matchLabels:
+              app: cpx-cic
           template:
             metadata:
-              name: cpx-cic
               labels:
                 app: cpx-cic
-              annotations:
+              name: cpx-cic
             spec:
-              serviceAccountName: citrix
               containers:
-                - name: cpx
-                  image: "quay.io/citrix/citrix-k8s-cpx-ingress:13.0-36.29"
-                  securityContext:
-                    privileged: true
-                  env:
-                  - name: "EULA"
-                    value: "yes"
-                  - name: "KUBERNETES_TASK_ID"
-                    value: ""
-                  ports:
-                  - containerPort: 80
-                    hostPort: 80
-                  - containerPort: 443
-                    hostPort: 443
-                  imagePullPolicy: Always
-                # Add cic as a sidecar
-                - name: cic
-                  image: "quay.io/citrix/citrix-k8s-ingress-controller:1.5.25"
-                  imagePullPolicy: Always
-                  env:
-                  - name: "EULA"
-                    value: "yes"
-                  - name: "NS_IP"
-                    value: "127.0.0.1"
-                  - name: "NS_PROTOCOL"
-                    value: "HTTP"
-                  - name: "NS_PORT"
-                    value: "80"
-                  - name: "NS_DEPLOYMENT_MODE"
-                    value: "SIDECAR"
-                  - name: "NS_ENABLE_MONITORING"
-                    value: "YES"
-                  - name: POD_NAME
-                    valueFrom:
-                      fieldRef:
-                        apiVersion: v1
-                        fieldPath: metadata.name
-                  - name: POD_NAMESPACE
-                    valueFrom:
-                      fieldRef:
-                        apiVersion: v1
-                        fieldPath: metadata.namespace
-                  args:
-                    - --default-ssl-certificate
-                      $(POD_NAMESPACE)/default-cert
-                  imagePullPolicy: Always
+              - env:
+                - name: EULA
+                  value: "yes"
+                - name: KUBERNETES_TASK_ID
+                image: quay.io/citrix/citrix-k8s-cpx-ingress:13.0-36.29
```

Comment:
RBAC has to be updated; 
CIC Image version has to be updated line 40 and line187 .